go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/containerd/v2
 
-go 1.23.0
+go 1.24.0
```

Comment:
I'm unsure about this, do we want to stay on 1.23?

Reply:
Yes, we need to stay on 1.23.  See https://github.com/kubernetes/kubernetes/issues/130799.